infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/PlanLayoutTransformation.kt

```diff
@@ -84,7 +84,9 @@ fun toTrackLayoutKmPosts(
     planToGkTransformation: ToGkFinTransformation,
 ): List<TrackLayoutKmPost> {
     return kmPosts.mapIndexedNotNull { _, kmPost ->
-        if (kmPost.location != null && kmPost.kmNumber != null) {
+        if (
+            kmPost.location != null && kmPost.kmNumber != null && (kmPost.location.x != 0.0 || kmPost.location.y != 0.0)
```

Comment:
Kirjoittaisin tuon sijaintiehdon ehkä toisin päin (`!(kmPost.location.x == 0.0 && kmPost.location.y == 0.0)`) sillä se on itselleni helpompi lukea, mutta melko puhdas makuasia. Loogisestihan tämä on täysin sama asia 👍

Reply:
Joo noi on vähän että kenelle helpompi ja minä päivänä mikäkin. Ymmärrettävyteen voisi varmaan myös vaikuttaa nimeämällä osan logiikkaketjusta muuttujaksi, mutta eipä tosin tässä erityisen suuri hyöty.

Reply:
Inhottaa aina nämä 0-vertailut koordinaatilla koska monessa koordinaatistossa 0,0 on ihan validi koordinaatti. Toisaalta näinhän se kuitenkin on että pakko tuo on tiputtaa kyydistä :shrug: Ehkä toinen tapa ajatella olisi että  meillä tavallaan se 1. pylväs on se oikea erikoisuus enemmänkin kuin tuo sijainti, mutta eipä se taida olla yhtään selkeämpää, varsinkin kun sekään ei aina ole rikkinäinen.